packages/dev/core/src/Collisions/gpuPicker.ts

```diff
@@ -222,6 +250,7 @@ export class GPUPicker {
         }
 
         this._cachedScene = scene;
+        this._engine = scene.getEngine();

```

Comment:
in this case let s call it cachedEngine for consistency ?

Reply:
I thought about to name it `chacedEngine` and about wy @deltakosh named it cached and not jiust scene. Cached here means that the scene has prepared materials specific for the picker and that we don't to create the materials again on consequent calss to the pick function. This doesn't apply to engine.